### DISCUSSION TITLE
Claude 3.5 accessibility

### DISCUSSION THREAD

**Discussion by michaelyeung445** [zed-industries/zed#17740] [Feature Requests]
I tried to use zed  preset ai (3.5 sonnet) but it said that this llm is not supported to my area because I am currently in Hong Kong, I tried to use VPN but still not work, why it is so restricted 😭

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/issues/16844#issuecomment-2322907358
